contract/src/lib.rs

```diff
@@ -64,6 +64,7 @@ pub struct ResharingContractState {
 
 #[derive(BorshDeserialize, BorshSerialize, Serialize, Deserialize, Debug)]
 pub enum ProtocolContractState {
+    NotInitialized,
```

Comment:
Why do we need one more status?

Reply:
It's a stop-gap for now, otherwise it's very hard to deploy multichain to GCP.

Reply:
I will document this behaviour